inference-engine/src/mkldnn_plugin/nodes/mkldnn_bin_conv_node.cpp

```diff
@@ -1081,14 +1081,14 @@ bool MKLDNNBinaryConvolutionNode::canFuse(const MKLDNNNodePtr& node) const {
         return false;
 
     // Binarization have to be last operation in fusing chain
-    if (isFusedWith(Quantize))
+    if (isFusedWith(FakeQuantize))
         return false;
 
-    if (node->getType() == Quantize) {
-        auto* quantizeNode = dynamic_cast<MKLDNNQuantizeNode*>(node.get());
-        if (quantizeNode == nullptr)
-            THROW_IE_EXCEPTION << "Cannot get quantize node " << node->getName();
-        return quantizeNode->isBinarization();
+    if (node->getType() == FakeQuantize) {
+        auto* fakeQuantizeNode = dynamic_cast<MKLDNNFakeQuantizeNode*>(node.get());
```

Comment:
You can remove dynamic cast since getAlgorithm is available

Reply:
removed